package.json

```diff
@@ -180,7 +180,7 @@
     "mochawesome-merge": "^4.2.1",
     "mochawesome-report-generator": "^6.0.1",
     "mock-apollo-client": "^0.4.0",
-    "prettier": "^1.19.1",
+    "prettier": "^2.7.1",
```

Comment:
OK, I can make a separate PR for that - but we need to upgrade `prettier` - our outdated version doesn't understand some of the TypeScript syntax, for example:

```ts
type MyUnion = 'apple' | 'orange'; 
type ChangedUnion = `prefix-${MyUnion}`
```

That's the reason I've updated prettier - the CI was failing because Prettier didn't understand some syntax 🙈